### PR TITLE
Foreman deployment

### PR DESCRIPTION
**Pull request by pitr-ch**
cc @GregSutcliffe @eLobato @shlomizadok @witlessbird
long running PR for the Foreman Deployment support
see https://github.com/theforeman/hammer-cli-foreman/pull/159

### CODE REVIEW

**Comment by lzap** [comment]
Wow, that's awesome. I wish bot commented syntax errors here as well ;-)

**Comment by pitr-ch** [comment]
cc @bbuckingham

File: doc/model/generate-svgs.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```
Comment (ekohl) [inline doc/model/generate-svgs.sh]: I think you should use `#!/bin/bash` here because `set -e` and `set -x` are bash features, but not 100% sure.

Comment (pitr-ch) [inline doc/model/generate-svgs.sh]: Changing to bash, thanks.

**Comment by pitr-ch** [comment]
Closing in favor of https://github.com/theforeman/foreman_deployments and
https://github.com/theforeman/hammer_cli_foreman_deployments.

**Pull request closed without merging**